src/Components/HouseholdDataBlock/HouseholdDataBlock.js

```diff
@@ -422,7 +428,7 @@ const HouseholdDataBlock = ({ handleHouseholdDataSubmit }) => {
     );
   };
 
-  const handleContinueSubmit = (event, validateInputFunction, inputToBeValidated, stepId, questionName, uuid) => {
+  const handleContinueSubmit = (event, uuid) => {
```

Comment:
This change breaks things because handleContinueSubmit is called by another function that passes in the extra parameters.
![image](https://github.com/Gary-Community-Ventures/benefits-calculator/assets/62856626/ea2787f7-bda5-4e25-b52f-ea62ee223650)